roles/config-routes/tasks/route.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+- name: "Configure route for the specific interface"
+  template:
+    src: route.j2 
+    dest: /etc/sysconfig/network-scripts/route-{{ route.device }}
+  with_items:
+  - '{{ routes }}'
+  loop_control:
+    loop_var: route
```

Comment:
Because that could cause an outage. Was going back/forth on that and didn't want to introduce the risk.